test/unit/go-version-parser.spec.ts

```diff
@@ -6,17 +6,23 @@ import { Elf } from "../../lib/go-parser/types";
 import { extractModulesFromBinary } from "../../lib/go-parser/version-parser";
 
 describe("go binary version parser", () => {
-  const files = fs.readdirSync(pathJoin(__dirname, "../fixtures/go-binaries"));
+  const files = fs.readdirSync(pathJoin(__dirname, "../fixtures/go-binaries"), {
+    withFileTypes: true,
+  });
   for (const file of files) {
+    if (!file.isFile()) {
+      continue;
+    }
```

Comment:
Not sure I understand why we need this if. If one of us accidentally added something strange to `../fixtures/go-binaries` I'm fine with the tests breaking in runtime 🤷‍♀️

Reply:
The tests already break because I've added a `symbols` folder, that's why we need it :-)

Reply:
Ah I see, alright then.